app/content/models/strike.py

```diff
@@ -36,10 +50,53 @@ class Meta:
     def __str__(self):
         return f"{self.user.first_name} {self.user.last_name} - {self.description} - {self.strike_size}"
 
+    def save(self, *args, **kwargs):
+        if self.created_at is None:
```

Comment:
Burde ikke brukeren få en notifikasjon uansett, eller har `created_at` en spesiell betydning her?

Reply:
denne er slik at brukeren ikke får notification om det skjer endringer i prikken. F.eks. endrer dato

Reply:
Er det da bedre å bruke `id`?

Reply:
var vist en bug der id ble satt, men created_at var null 🤷🏾

Reply:
Skjønner, det kan se ut til at [django håndterer UUID'en](https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.UUIDField) i stedet for databasen